.github/CODEOWNERS

```diff
@@ -14,7 +14,7 @@
 /src/Hosting/                   @tratcher @anurse
 /src/Http/                      @tratcher @jkotalik @anurse
 /src/Middleware/                @tratcher @anurse
-/src/ProjectTemplates/          @ryanbrandenburg
+# /src/ProjectTemplates/          @ryanbrandenburg
```

Comment:
Remove, don't comment out

Reply:
The comment is done to keep things [in sync with master](https://github.com/dotnet/aspnetcore/blob/master/.github/CODEOWNERS#L19). I seem to remember that I originally was going to remove the entry but @mkArtakMSFT suggested I keep it as a comment to make sure it was still clear who "owned" the area.

@mkArtakMSFT am I remembering that correctly and/or what are your current opinions?

Reply:
Preference is **always** to remove lines rather than comment them out. If you want to add a clear comment like `# @ryanbrandenburg loves ProjectTemplates but not email notifications about it` that's fine but don't leave it looking like the "real" lines